test/built-ins/BigInt/asIntN/arithmetic.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
```

Comment:
we already have an esid: `sec-integer.asintn`

https://tc39.github.io/proposal-bigint/#sec-integer.asintn